packages/node/test/domain.test.ts

```diff
@@ -1,6 +1,10 @@
 import { getCurrentHub, Hub } from '@sentry/core';
 import * as domain from 'domain';
 
+import * as Sentry from '../src';
+
+console.log(Sentry.SDK_NAME);
```

Comment:
Debug leftover.

Reply:
Pushed another commit, we need the import and without usage, tslint complains.